queries.sql

```diff
@@ -7,3 +7,33 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals
+ROLLBACK TRANSACTION;
+
```

Comment:
- [Optional] Dear @RitaDee, It's great you update the changes and see the result. But the only way to make sure it persists/revert is to have a select statement next to the `ROLLBACK TRANSACTION` statement.

Reply:
Noted with so many thanks @TadesseDev.